packagelist

```diff
@@ -299,6 +299,7 @@ linux-image-unsigned-stable-deb
 linux-kernel
 linux-kernel-stable
 linux-libc-dev-okpine-deb
+linux-longterm-6.6-headers-modular-deb
```

Comment:
drop `-modular` from the naming of the package

Reply:
Actually the other one needs to be changed too. 
They should be:
`linux-headers-lts-6.6-deb` and `linux-modules-lts-6.6-deb`

Reply:
> Actually the other one needs to be changed too. They should be: `linux-headers-lts-6.6-deb` and `linux-modules-lts-6.6-deb`
> 
> FWD from discord: So in the end based on how Ubuntu packages mainline kernel, your pacscripts should be:
> 
> ```
> linux-headers-lts-6.6-deb
> linux-headers-generic-lts-6.6-deb
> linux-image-unsigned-lts-6.6-deb
> linux-modules-lts-6.6-deb
> ```

So, the merge map should be:

`linux-modules-lts-6.6-deb` -> `linux-headers-lts-6.6-deb` -> `linux-headers-generic-lts-6.6-deb` -> `linux-image-unsigned-lts-6.6-deb`

Like that, right ?

Reply:
There’s no reason why they can’t all be uploaded at once. See how `linux-kernel` and `linux-kernel-stable` are packaged for reference, you could create `linux-kernel-lts-6.6` meta package and do them all in one go

Reply:
> There’s no reason why they can’t all be uploaded at once. See how `linux-kernel` and `linux-kernel-stable` are packaged for reference, you could create `linux-kernel-lts-6.6` meta package and do them all in one go

So, what should I do know for this commit and the last commit before for `modules`? _I am still confused because I replied this in the waking hours_

Reply:
Continue on Discord for replying on it

Reply:
Got it, it's fixed. Please accept it when `pre-commit` get the job done.